lib/test/Test/Pos/Txp/Toil/UtxoSpec.hs

```diff
@@ -266,6 +270,13 @@ scriptTxSpec = describe "script transactions" $ do
                 alwaysSuccessValidator
                 (const witness)
 
+        it "validator script provided in witness is greater \
+           \than adopted script version" $ do
+            let witness = ScriptWitness intValidatorWithVersionOne goodIntRedeemer
+            txShouldFailWithInvalidWitness $ checkScriptTx
+                alwaysSuccessValidator
```

Comment:
AFAIU, you should pass `intValidatorWithVersionOne` here.

Reply:
Yes that is correct